components/application-mgt/org.wso2.carbon.identity.application.mgt/pom.xml

```diff
@@ -29,6 +29,10 @@
     <url>http://wso2.org</url>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
```

Comment:
why is this needed?

Reply:
As `StringUtils.isAnyBlank()` method is used.

Reply:
Removed this dependency as we are no longer using the mentioned method.